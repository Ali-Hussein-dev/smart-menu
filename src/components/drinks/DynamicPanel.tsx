// import DrinkItem from './DrinkItem'

//=======================
const DynamicPanel: React.FC = ({ children }) => {
  // hooks
  //--------------------------------------
  // functions
  //--------------------------------------
  return <div className="">{children}</div>
}
export default DynamicPanel
