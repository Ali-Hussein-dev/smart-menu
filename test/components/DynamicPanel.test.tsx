import DynamicPanel from '@/components/drinks/DynamicPanel'
import { render } from '../testUtils'

describe('Dynamic Panel', () => {
  const { container } = render(<DynamicPanel>children</DynamicPanel>)
  test('matches snapshot', () => {
    expect(container).toMatchSnapshot()
  })
})
