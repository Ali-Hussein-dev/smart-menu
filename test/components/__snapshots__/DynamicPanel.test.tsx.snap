// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dynamic Panel matches snapshot 1`] = `
<div>
  <div
    class=""
  >
    children
  </div>
</div>
`;
